root/tree/index/runindex64.C

```diff
@@ -25,20 +26,22 @@ int runindex64(){
   tree->Branch("run", &run, "run/l");
   tree->Branch("event", &event, "event/l");
 
-  ULong64_t events[] = { 1,2,3, bigval, biguval, 5 };
-  run = 5;
-  for(int i=0; i<sizeof(events)/sizeof(*events); i++){
+  ULong64_t   runs[] = { 8,5,5,5,      5,       5,  0,      4, 6, biguval,  bigval, bigval, biguval};
+  ULong64_t events[] = { 0,1,3,2, bigval, biguval,  5, bigval, 3,  bigval, biguval, bigval, biguval};
+  for(size_t i=0; i<sizeof(events)/sizeof(*events); i++){
+    run = runs[i];
     event = events[i];
     tree->Fill();
   }
-  run = 4; event = bigval; tree->Fill();
-  run = 6; event = 3; tree->Fill();
-  run = biguval; event = bigval; tree->Fill();
   tree->Write();
+  tree->Scan("run:event","","colsize=30");
```

Comment:
What is the intent of this addition? (I.e. what is it testing?)

Reply:
It shows that Scan works well, but somehow the entry is not foundnin Mac, even if the Scan part works well

Reply:
Good, I guess.  This means this is a debugging tool and is not inherent to the test.  To keep the output platform independent we could run the `Scan` only in the cases where a failure is detected.